recipes/tableone/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "tableone" %}
+{% set version = "0.4.8" %}
+{% set sha256 = "63c6d55a7bc0010ced0a6b1e88f295ce41fd93d9f8a473e14cf1c760c93a27b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - pandas
+    - numpy
+    - scipy
```

Comment:
I think that most of these are not build dependencies (one can install the code without these (although maybe not run the code))